From 4ccde08493ccd4f17b180a2a283e3e924a6311e9 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 20 Apr 2021 09:32:35 +0530
Subject: [PATCH] Update health HAL changes based on S

This patch updates the changes based on Android 12
1. system/core/base is moved to system/libbase
2. Health hal 1.0 is deprecated so IHealth.h is replaced
with types.h.

Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 health/2.1/hal_conversion.h      | 2 +-
 health/2.1/libhealthd/Android.bp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/health/2.1/hal_conversion.h b/health/2.1/hal_conversion.h
index f4b4eae..1b94bbe 100644
--- a/health/2.1/hal_conversion.h
+++ b/health/2.1/hal_conversion.h
@@ -15,7 +15,7 @@
  */
 #ifndef HARDWARE_INTERFACES_HEALTH_V1_0_DEFAULT_INCLUDE_HAL_CONVERSION_H_
 #define HARDWARE_INTERFACES_HEALTH_V1_0_DEFAULT_INCLUDE_HAL_CONVERSION_H_
-#include <android/hardware/health/1.0/IHealth.h>
+#include <android/hardware/health/1.0/types.h>
 #include <healthd/healthd.h>
 namespace android {
 namespace hardware {
diff --git a/health/2.1/libhealthd/Android.bp b/health/2.1/libhealthd/Android.bp
index eaecc31..4235db9 100644
--- a/health/2.1/libhealthd/Android.bp
+++ b/health/2.1/libhealthd/Android.bp
@@ -6,7 +6,7 @@ cc_library_static {
     vendor_available: true,
     recovery_available: true,
     cflags: ["-Werror"],
-    include_dirs: ["system/core/base/include"],
+    include_dirs: ["system/libbase/include"],
     header_libs: ["libhealthd_headers"],
     shared_libs: [
         "libcutils",
-- 
2.17.1

